content/Dev/remote_settings_history.rst

```diff
@@ -0,0 +1,178 @@
+
+The question is legitimate, and we could defend our pipeline because we have a validated security workflow, with VPN access, permissions management, some review and signoff features, push notifications, diff-based synchronization, content signature verification...
+
+Nevertheless, at the same time, more and more use-cases are server-to-server or have automated publication from scripts. And some even built their own UI.
+
+Maybe our *Remote Settings* service could be achieved with just something like a Git repo, static data, and a CDN in front.
+
+Clients pull blobs and content signatures from a URL, each use-case being responsible of parsing these binaries as JSON or using them as plain resources or files. Permissions setup could be achieved using protected branches and submodules in the repo. No more UI to maintain, the reviewing process would just happen upfront, before merging the pull-request. No more backend and database with records and fields, just a few scheduled tasks or commit hooks.
+
+A dumb pipe of static data, with complex workflows outside the system itself. A little bit `like this blog </publish-your-pelican-blog-on-github-pages-via-travis-ci.html>`_ actually.
+
+Why not!
+
+
+Thoughts
+========
+
+I hope this long article helped you understand how a long term project can evolve and mutate. I thought it would be interesting to see it from within a company like Mozilla, often mentioned in headlines.
+
+
+What made the success?
+----------------------
+
+*Remote Settings* became a critical part of Firefox. Clearly, since it is leveraging only a subset of *Kinto*, the success can be largely attributed to the vision and efforts of our early team. We're still friends and I'm super proud of what we accomplished! Big up!
+
+The patience of stakeholders who held our hand in order to land patches in Firefox massively contributed to the success. Special thanks to `Gijs <https://github.com/gijsk>`_, `Lina <https://github.com/linacambridge>`_, `Florian <http://blog.queze.net/>`_, `mgoodwin <https://github.com/computerist>`_, `Standard8 <https://github.com/Standard8>`_...
+
+Another factor is that we were consistently protecting the functional scope of the system. It is a data pipe: publish data on one side, reach the target audience reliably. It took a lot of effort to say «no», and keep the complexity trend downwards, instead of building dedicated features for specific use-cases.
+
+During our company meetings, we were making sure we would spend more time with people, and the least in front of computers. I don't cross the world to do what I can do from home! We had joyful moments and this happiness certainly contributed to the success :) Through the numerous use-cases, I got to know many teams in the company, and always worked in a fantastic atmosphere. I am happy to help, they seem happy with the solution, it is great!
```

Comment:
I think this understates how important our first customers were to the growth and adoption of RS.  The security/crypto teams and the Activity Stream (now UJET) deserve much credit.  Their advocacy for remote-settings and working closely with us to make it better made a big difference.  

Also I think it worth mentioning that taking a more "product and customer first" perspective, combined with extra mile customer service (you wrote many early implementations for other teams!) were also crucial in adoption.

Reply:
I completely agree! They clearly deserve a lot of recognition!